### DISCUSSION TITLE
Source of Hydrogen demos

### DISCUSSION THREAD

**Discussion by jecuendet** [hydrogen-music/hydrogen#1660] [General]
Hi,
I would like to get the source files of the Hydrogen demos found here:
https://sourceforge.net/projects/hydrogen/files/Hydrogen/demos/

Are they available?
Thanks

**Comment by theGreatWhiteShark** [comment]
Hmm. Those look like user contributions and I couldn't find any resource files holding the same names. After all they were added almost 20 years ago.

But there are two other sources for demo songs and patterns
- https://github.com/hydrogen-music/Song-and-pattern-repository
- https://github.com/hydrogen-music/hydrogen/tree/master/data/demo_songs

Hopefully they will be of help.
